src/reports/ChartReportPage.jsx

```diff
@@ -33,6 +33,7 @@ const ChartReportPage = () => {
   const [items, setItems] = useState([]);
   const [types, setTypes] = useState(['speed']);
   const [type, setType] = useState('speed');
+  const [timeIndex, setTimeIndex] = useState('fixTime');
```

Comment:
I don't think `timeIndex` is a good name. Maybe `timeType` or something.

Reply:
Agreed. it was actually timeType before i changed it to match the name in the UI 😆

I'll change it shortly